frontend/app/routes/$lang+/_protected+/applications+/index.tsx

```diff
@@ -53,33 +56,64 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   auditService.audit('page-view.applications', { userId: idToken.sub });
   instrumentationService.countHttpStatus('applications.view', 200);
 
-  return json({ applications, meta });
+  return json({ applications, meta, sortOrder });
 }
 
 export default function ApplicationsIndex() {
+  const [, setSearchParams] = useSearchParams();
   const { t } = useTranslation(handle.i18nNamespaces);
-  const { applications } = useLoaderData<typeof loader>();
+  const { applications, sortOrder } = useLoaderData<typeof loader>();
   const params = useParams();
   const userOrigin = useUserOrigin();
 
-  //TODO: will update <InlineLink rounteId> when view details page is ready
+  function handleOnSortOrderChange(e: ChangeEvent<HTMLSelectElement>) {
+    setSearchParams((prev) => {
+      prev.set('sort', e.target.value);
+      return prev;
+    });
+  }
+
   return (
     <>
+      <p>{t('applications:index.page-title')}</p>
```

Comment:
We can remove this text. The page title is already rendered as an `<h1>`.

Reply:
Fixed: 1e363b7515ed32ceb9084798bea09b9f427b6b0b.